docs/operations/ces_installation_en.md

```diff
@@ -1,19 +1,408 @@
+  },
+  "dogus": {
+    "defaultDogu": "cas",
+    "install": [
+      "official/ldap",
+      "official/postfix",
+      "k8s/nginx-static",
+      "k8s/nginx-ingress",
+      "official/cas"
+    ],
+    "completed": true
+  },
+  "admin": {
+    "username": "admin",
+    "mail": "admin@admin.admin",
+    "password": "changeme",
+    "adminGroup": "cesAdmin",
+    "completed": true,
+    "adminMember": true,
+    "sendWelcomeMail": false
+  },
+  "userBackend": {
+    "dsType": "embedded",
+    "server": "",
+    "attributeID": "uid",
+    "attributeGivenName": "",
+    "attributeSurname": "",
+    "attributeFullname": "cn",
+    "attributeMail": "mail",
+    "attributeGroup": "memberOf",
+    "baseDN": "",
+    "searchFilter": "(objectClass=person)",
+    "connectionDN": "",
+    "password": "",
+    "host": "ldap",
+    "port": "389",
+    "loginID": "",
+    "loginPassword": "",
+    "encryption": "",
+    "completed": true,
+    "groupBaseDN": "",
+    "groupSearchFilter": "",
+    "groupAttributeName": "",
+    "groupAttributeDescription": "",
+    "groupAttributeMember": ""
+  }
+}
+```
+
+#### Installation
+
+Start the installation with the aforementioned `./install.sh`.
+
+The setup will start automatically if `completed: true` is in each section of `setup.json`.
+Otherwise the setup can be started manually:
+
+`curl -I --request POST --url http://<any-node-ip>:30080/api/v1/setup`
+
+> Info: If the setup process aborts because an invalid value was specified in `setup.json`, the configmap `k8s-setup-config` must be deleted after correcting the `setup.json`.
+> After this, the setup can be started again.
+
+The Cloudogu EcoSystem can be deleted **completely** from the cluster with the following commands (the created registry credentials remain unaffected):
+
+- Delete dogus
+```bash
+kubectl delete dogus -l app=ces -n ecosystem
+```
+
+- Delete remaining resources
+```bash
+kubectl patch cm tcp-services -p '{"metadata":{"finalizers":null}}' --type=merge -n ecosystem || true \
+&& kubectl patch cm udp-services -p '{"metadata":{"finalizers":null}}' --type=merge -n ecosystem || true \
+&& kubectl delete statefulsets,deploy,secrets,cm,svc,sa,rolebindings,roles,clusterrolebindings,clusterroles,cronjob,pvc,pv --ignore-not-found -l app=ces -n ecosystem
+```
+
+## 4. notes for different infrastructures and cloud providers
+
+### Using mirrored registries
+
+If mirrored registries are used, it is quite possible that all Docker images are located in a
+subproject in the registry (here, for example, `organization`).
+
+Example structure:
+```
+example.com/
+├── organization <-
+│   ├── k8s
+│   │   ╰── k8s-dogu-operator
+│   │       ╰── 0.1.0
+│   ├── official
+│   │   ╰── cas
+│   │       ╰── 0.1.0
+│   ├── premium
+│   ╰── other namespace
+```
+
+In this case, a rewrite must be created for the container configuration of `k3s` so that images
+such as `example.com/longhorn/manager` can be obtained from `example.com/organization/longhorn/manager`.
```

Comment:
```suggestion
In this case, a rewrite must be created for the container configuration of `k3s` so that images
such as `example.com/longhorn/manager` can be obtained from e.g. `example.com/organization/longhorn/manager`.
```

Reply:
done